### ISSUE TITLE
Hi!

### ISSUE THREAD

**Issue by cuzoxeto** [GrowtopiaNoobs/GrowtopiaServer#36]
Hi GrowtopiaNoobs,
I tried to communicate with you, but couldn't find your email or Discord.
Please there is a emergency about your code and development.
@GrowtopiaNoobs

**Comment by HellixBG** [comment]
I don't see the emergency about the code

**Comment by cuzoxeto** [comment]
No, it's not about that.
It's about one of his code videos and somebody stole the code and running their server but code is under APGL, they need to release the source code too but they didn't.

Also I need to say something important.

**Comment by GrowtopiaNoobs** [comment]
@cuzoxeto Sure you can contact me on growtopianoobs@gmail.com or Anybody#8639.

**Comment by GrowtopiaNoobs** [comment]
@FocusedSG Yes. This is requested by license and everyone should follow it. But there isn't anything what I can do when anyone breaks the license, so it is getting kinda useless. Mostly all of projects on github have some license, which you should follow (attribute/dont attribute author, release modifications of source under some conditions, who is owner of code, etc).
